### ISSUE TITLE
Mag Block / Brick Range (suggestion)

### ISSUE THREAD

**Issue by cloudrambler** [tsunamayo/Starship-EVO#2908]
Presently the mag block (which is rather superb) does not have any indication of range or alignment. Could we have a visual corridor effect like the laser sensor and / or an indicator light on the block to show that a suitable object is within range.

If adding a visual corridor effect, it would be nice to have a toggle option (f: context menu) to hide it again.

**Comment by tsunamayo** [comment]
Yes I agree I wanted to do this, it was just a time issue. Still on the to-do list. Thanks
